client/db/bolt/db.go

```diff
@@ -174,9 +175,9 @@ func (db *BoltDB) Store(k string, v []byte) error {
 	})
 }
 
-// ValueExists checks if a value was previously stored in the general-use
-// bucket at the specified key.
-func (db *BoltDB) ValueExists(k string) (bool, error) {
+// IsEncryptionParamsSet checks whether encryption parameters associated with key k were previously stored in DB
+// via a call to SetEncryptionParams.
```

Comment:
There is a effort to keep comments under 80 columns in width from the left. It's not set in stone, but this comment  and others go too far to the right.

Reply:
Fixed.

Although, I would point out:
- usually, linters work better to enforce such conventions

Reply:
There are valid exceptions to this convention, so it's not great for a linter.  Please just wrap such comments.  Setup something in your editor and you'll never have to think about it.

Reply:
Okay, np.